.eslintrc.json

```diff
@@ -76,7 +76,7 @@
     "require-yield": "off",
     "prefer-template": "warn",
     "no-underscore-dangle": "off",
-    "comma-dangle": ["warn", "never"],
+    "comma-dangle": ["warn", "only-multiline"],
```

Comment:
This feels like this warrants a team discussion before including it without comment.    Not saying I disagree, but we should team-agree on this.

Reply:
Yep, sorry. I was experimenting and forgot to undo before commit. I will revert.